### PR TITLE
Add MyFTSO

### PR DESCRIPTION
**Pull request by virtumbe**
Add MyFTSO

### CODE REVIEW

**Review by perlindstroem — CHANGES_REQUESTED** [review]
A few comments;
* The icon is not centered in the image, please fix.
* Please optimize the image with https://tinypng.com/.
* You are currently using the same address for Flare and Songbird, the best practice and recommendation is to use separate addresses, however we can accept your current setup if you want.

**Comment by virtumbe** [comment]
> A few comments;
> 
> * The icon is not centered in the image, please fix.
> * Please optimize the image with https://tinypng.com/.
> * You are currently using the same address for Flare and Songbird, the best practice and recommendation is to use separate addresses, however we can accept your current setup if you want.

Did all the requested changes. Thanks in advance.

**Review by perlindstroem — CHANGES_REQUESTED** [review]
Great! A side-effect is that you need to have a separate copy of the image with the new address in the URI, we require one image per address.

**Comment by virtumbe** [comment]
> Great! A side-effect is that you need to have a separate copy of the image with the new address in the URI, we require one image per address.

Sorry, just uploaded the requested file!

File: bifrost-wallet.providerlist.json [hunk 1]
```diff
@@ -1638,6 +1638,22 @@
...
+			"address": "0x399a2de69e38d93bc397eb2b1f5487bd25a71c00",
+			"logoURI": "https://raw.githubusercontent.com/TowoLabs/ftso-signal-providers/master/assets/0x399a2de69e38d93bc397eb2b1f5487bd25a71c00.png"
+		},
+		{
+			"chainId": 14,
+			"name": "MyFTSO",
+			"description": "MyFtso is a community centric data provider. We are looking to deliver accurate and reliable price feeds to the Flare/Songbird Networks to maximize returns for all.",
+			"url": "https://myftso.xyz",
+			"address": "0x6da8b10612c2C5a5A62642d4a666bC830E74FC40",
+			"logoURI": "https://raw.githubusercontent.com/TowoLabs/ftso-signal-providers/master/assets/0x399a2de69e38d93bc397eb2b1f5487bd25a71c00.png"
```
Comment (perlindstroem) [inline bifrost-wallet.providerlist.json:1656]: I see that you uploaded a new image, great! However, this row also needs to mirror that update.

**Review by perlindstroem — CHANGES_REQUESTED** [review]

**Review by perlindstroem — APPROVED** [review]

**Pull request merged**